tests/system/test_marathon_universe.py

```diff
@@ -36,15 +36,17 @@ def test_install_marathon():
     """
 
     # Install
-    @retrying.retry(wait_fixed=1000, stop_max_attempt_number=30, retry_on_exception=DCOSException)
+    @retrying.retry(wait_fixed=1000, stop_max_attempt_number=30,
+                    retry_on_exception=lambda e: isinstance(e, DCOSException))
```

Comment:
Should it be just `common.ignore_exception`?

Reply:
I am actually not sure, @kensipe?

Reply:
the `common.ignore_exception` will ignore all exceptions... the way this is written it only retry on a `DCOSException`... I can see cases where that is what is desired.    Is there a reason to be specific here?   should we modify our common code to allow for this case?

Reply:
Lol, originally I thought you introduced the retry on `DCOSException` here but git blame says it was me 🙈.